desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTabContents/StorageBrowserTabContent.tsx

```diff
@@ -79,6 +156,37 @@ const StorageBrowserTabContent: React.FC<StorageBrowserTabContentProps> = ({
             showIcon={false}
           />
         </div>
+        <div className="hue-storage-browser__actions-bar">
+          <Input className="hue-storage-browser__search" placeholder={t('Search')} />
+          <div className="hue-storage-browser__actions-bar-right">
+            <Dropdown
+              overlayClassName="hue-storage-browser__actions-dropdown"
+              menu={{
+                items: bulkActionsMenuItems,
+                className: 'hue-storage-browser__action-menu'
+              }}
+              trigger={['hover', 'click']}
+            >
+              <Button className="hue-storage-browser__bulk-action-btn">
+                {t('Bulk Actions')}
+                <DropDownIcon />
+              </Button>
+            </Dropdown>
+            <Dropdown
+              overlayClassName="hue-storage-browser__actions-dropdown"
+              menu={{
+                items: newActionsMenuItems,
+                className: 'hue-storage-browser__action-menu'
+              }}
+              trigger={['hover', 'click']}
+            >
+              <Button className="hue-storage-browser__new-btn" icon={<PlusCircleIcon />}>
```

Comment:
Just a NIT: I noticed the icon isn't properly aligned in the screenshots.

Reply:
Will make the necessary changes in the following PR